sdk/search/ai-search/rollup.base.config.js

```diff
@@ -0,0 +1,121 @@
+import path from "path";
+import nodeResolve from "@rollup/plugin-node-resolve";
+import multiEntry from "@rollup/plugin-multi-entry";
+import cjs from "@rollup/plugin-commonjs";
+import replace from "@rollup/plugin-replace";
+import { terser } from "rollup-plugin-terser";
+import sourcemaps from "rollup-plugin-sourcemaps";
+import viz from "rollup-plugin-visualizer";
+
+const pkg = require("./package.json");
+const depNames = Object.keys(pkg.dependencies);
+const devDepNames = Object.keys(pkg.devDependencies);
+const input = "dist-esm/src/index.js";
+const production = process.env.NODE_ENV === "production";
+
+export function nodeConfig(test = false) {
+  const externalNodeBuiltins = ["events"];
+  const baseConfig = {
+    input: input,
+    external: depNames.concat(externalNodeBuiltins),
+    output: { file: "dist/index.js", format: "cjs", sourcemap: true },
+    preserveSymlinks: false,
+    plugins: [
+      sourcemaps(),
+      replace({
+        delimiters: ["", ""],
+        values: {
+          // replace dynamic checks with if (true) since this is for node only.
+          // Allows rollup's dead code elimination to be more aggressive.
+          "if (isNode)": "if (true)"
+        }
+      }),
+      nodeResolve({ preferBuiltins: true }),
+      cjs()
+    ]
+  };
+
+  if (test) {
+    // Entry points - test files under the `test` folder(common for both browser and node), node specific test files
+    baseConfig.input = ["dist-esm/test/*.spec.js", "dist-esm/test/node/*.spec.js"];
+    baseConfig.plugins.unshift(multiEntry({ exports: false }));
+
+    // different output file
+    baseConfig.output.file = "dist-test/index.node.js";
+
+    // mark devdeps as external
+    baseConfig.external.push(...devDepNames);
+
+    // Disable tree-shaking of test code.  In rollup-plugin-node-resolve@5.0.0, rollup started respecting
+    // the "sideEffects" field in package.json.  Since our package.json sets "sideEffects=false", this also
+    // applies to test code, which causes all tests to be removed by tree-shaking.
+    baseConfig.treeshake = false;
+  } else if (production) {
+    baseConfig.plugins.push(terser());
+  }
+
+  return baseConfig;
+}
+
+export function browserConfig(test = false) {
+  const baseConfig = {
+    input: input,
+    output: {
+      file: "browser/azure-ai-search.js",
```

Comment:
>browser [](start = 13, length = 7)

I believe Daniel just moved/renamed `browser/` folder to `dist-browser/` for the browser build output